docs/docs/Components/components-logic.md

```diff
@@ -241,6 +216,80 @@ Your flow should now look like this:
 |--------------|-------------|---------------------------------------------------------------|
 | run_outputs  | A `List` of types `Data`, `Message,` or `DataFrame`  | All outputs are generated from running the flow.                   |
 
+## Legacy components
+
```

Comment:
You should not have a heading without any body text. Add something to describe what this section is about.